### PR TITLE
teams/tidb: a vote for sylzd as committer

### PR DESCRIPTION
**Pull request by bb7133**
As is stated in the title.

This commit also fixes a typo of the vote record('votes/0592-tisonkun-as-tidb-committer.md').

### CODE REVIEW

**Comment by bb7133** [comment]
Hi @pingcap/tidb-maintainers , PTAL

File: votes/0611-sylzd-as-tidb-committer.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+[@sylzd](https://github.com/sylzd) has been promoted to the reviewer of TiDB on October(/votes/0550-sylzd-as-tidb-reviewer.md).
+
+After that, @sylzd has actively contributed to a variety of components of TiDB-server like 'DDL', 'expression', 'executor', 'parser' and 'dumpling'. He made a series of contributions for the new feature '[Placement Rule in SQL](https://docs.pingcap.com/tidb/stable/placement-rules-in-sql)' and fixed a lot of bugs of TiDB.
+
+IMO the work shows the contribution of @sylzd, as well as his sense of responsibility to `pingcap/tidb` project, as well as the TiDB community. The details of all his PRs and reviews are:
+
+* [40+ authored pull requests](https://github.com/pingcap/tidb/commits?author=sylzd).
+* [10+ reviewed pull requests](https://github.com/pingcap/tidb/pulls?q=is%3Apr+reviewed-by%sylzd)
+
+I (@bb7133) hereby nominate @tisonkun as TiDB committer and call for a vote.
```
Comment (kennytm) [inline votes/0611-sylzd-as-tidb-committer.md:14]:
```suggestion
I (@bb7133) hereby nominate @sylzd as TiDB committer and call for a vote.
```

**Review by tiancaiamao — APPROVED** [review]

**Review by disksing — APPROVED** [review]

**Pull request merged**